src/inference/dev_api/cpp_interfaces/interface/ie_iplugin_internal.hpp

```diff
@@ -441,4 +442,4 @@ convert_plugin(const std::shared_ptr<InferenceEngine::IInferencePlugin>& from);
  */
 #define IE_DEFINE_EXTENSION_CREATE_FUNCTION_DECLARATION(_IE_CREATE_EXTENSION_FUNC) \
     INFERENCE_EXTENSION_API(void)                                                  \
-    _IE_CREATE_EXTENSION_FUNC(::InferenceEngine::IExtensionPtr& ext)
+    _IE_CREATE_EXTENSION_FUNC(std::vector<::ov::Extension::Ptr>& ext)
```

Comment:
those are old API headers, why do we need to update them?

Reply:
@ilya-lavrenov  Yes, u are right, this is a old header for 1.0, but follow the ov::plugin upgrade style, mixed ov::extension in this file for new. You can see the plugin create func definition here:
https://github.com/openvinotoolkit/openvino/blob/9638b899db494ab7f8424191b15684f598a9f89a/src/inference/dev_api/cpp_interfaces/interface/ie_iplugin_internal.hpp#L435-L438
For now, I think this changes only for ov::extension will be fine at temporary. After this, another PR will be created to focus on the ov::plugin & ov::extension create function definition isolate for 2.0 & a new file will be created to replace this.